src/vt/vrt/collection/balance/model/load_model.h

```diff
@@ -215,6 +215,17 @@ struct LoadModel
    */
   virtual void updateLoads(PhaseType last_completed_phase) = 0;
 
+  /**
+   * \brief Provide all the comm info for a given phase
+   *
+   * \param[in] when the interval in which comm is desired
+   *
+   * \return the comm info
+   */
+  virtual CommMapType getComm([[maybe_unused]] PhaseOffset when) const {
+    return CommMapType{};
+  }
```

Comment:
Check if this can be made pure virtual:
```suggestion
  virtual CommMapType getComm([[maybe_unused]] PhaseOffset when) const =0;
```

Reply:
Let's leave it as is, there are some stub models that don't implement it.